alarmbot/src/ui/get_recent_summary.py

```diff
@@ -0,0 +1,88 @@
+import requests
+from datetime import datetime
+from .utils import CONSTANTS, get_user_info
+
+
+def retrieve_stats(data, username, tag):
```

Comment:
NIT: name is misleading to me. I would expect a stats object (maybe some class), but this returns a message. Not important.